python/katana/local/_graph.pyx

```diff
@@ -481,6 +531,182 @@ cdef class GraphBase:
+        return manager.ListAtomicTypes()
+
+    def has_node_entity_type(self, entity_type_id: int):
+        """
+        Return true iff an entity type `entity_type_id` exists for nodes
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        return manager.HasEntityType(entity_type_id)
+
+    def has_edge_entity_type(self, entity_type_id: int):
+        """
+        Return true iff an entity type `entity_type_id` exists for edges
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetEdgeTypeManager()
+        return manager.HasEntityType(entity_type_id)
+
+    def get_node_entity_type_id(self, str name):
+        """
+        Return the EntityTypeID for an atomic type with `name` for nodes
+        Return None if `name` is not an atomic node type
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        cdef string input_str = bytes(name, "utf-8")
+        if not manager.HasAtomicType(input_str):
+            return None
+        return manager.GetEntityTypeID(input_str)
+
+    def get_edge_entity_type_id(self, str name):
+        """
+        Return the EntityTypeID for an atomic type with `name` for edges
+        Return None if `name` is not an atomic edge type
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetEdgeTypeManager()
+        cdef string input_str = bytes(name, "utf-8")
+        if not manager.HasAtomicType(input_str):
+            return None
+        return manager.GetEntityTypeID(input_str)
+
+    def node_entity_type_to_type_name_set(self, entity_type_id: int):
+        """
+        Return a set of strings representing the names of the types for the type ID
+        `entity_type_id` for nodes
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        return handle_result_TypeNameSet(manager.EntityTypeToTypeNameSet(entity_type_id))
+
+    def edge_entity_type_to_type_name_set(self, entity_type_id: int):
+        """
+        Return a set of strings representing the names of the types for the type ID
+        `entity_type_id` for edges
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetEdgeTypeManager()
+        return handle_result_TypeNameSet(manager.EntityTypeToTypeNameSet(entity_type_id))
+
+    def get_atomic_node_type_name(self, entity_type_id: int):
+        """
+        Return the type name if `entity_type_id` is a valid node entity type ID. Otherwise,
+        return None
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        r =  manager.GetAtomicTypeName(entity_type_id)
+        if r:
+            return r.value()
+        return None
+
+    def get_atomic_edge_type_name(self, entity_type_id: int):
```

Comment:
These are not needed because we will have methods that return dicts so users can just to `entity_type_manager.by_id[id].name`. I think this is true of most of the single element accessors.